mariadb_skysql/hatch.toml

```
@@ -0,0 +1,4 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.9"]
```

The test failure is due to not having added any e2e tests. It's fine to not add them, but it needs to be specified here, so the following should clear that test failure:

```suggestion
python = ["3.9"]

[envs.default]
e2e-env = false
```